proctor-common/src/main/java/com/indeed/proctor/common/PayloadType.java

```diff
@@ -65,6 +66,14 @@ public enum PayloadType {
             "Object.<string, Object>",
             "{}",
             "getMap",
+            false),
+
+    JSON(
+            "json",
+            "JsonNode",
+            "Object",
+            "{}",
+            "getJson",
             false);
```

Comment:
You passed `isArraysType = false` for `JSON` though `JSON` can be array. Can't it be a problem in other places?

Reply:
I believe the isArraysType is specifically only for Java array classes (String[], Long[], Double[]) rather than the JsonNode class.  Similar to how a Map can have an array as the value, the JsonNode can have an array as the value also but is not an array.